experiments/conductor/cmd/runner/crd_generator_commands.go

```diff
@@ -221,32 +221,47 @@ func generateTypesAndMapper(opts *RunnerOptions, branch Branch) {
 		if err != nil {
 			log.Fatal(err)
 		}
+		hasMapperChange = true
+	} else {
+		log.Printf("SKIPPING generating mappers, %s already exists", mapperDir)
+	}
 
-		// Run goimports
-		cfg = CommandConfig{
-			Name: "Format generated code",
-			Cmd:  "go",
-			Args: []string{
-				"run", "-mod=readonly",
-				"golang.org/x/tools/cmd/goimports@latest",
-				"-w",
-				mapperDir,
-			},
+	// Run fix-gofmt script if any changes were made
+	if hasAPIChange || hasMapperChange {
+		// Run goimports on the generated code
+		cfg := CommandConfig{
+			Name:       "Fix imports",
+			Cmd:        "go",
+			Args:       []string{"run", "-mod=readonly", "golang.org/x/tools/cmd/goimports@latest", "-w", mapperDir},
```

Comment:
should we really on `make fmt`? or the existing `verify-goimports` ?

Reply:
Sorry could you elaborate?